ninfo/util.py

```diff
@@ -25,25 +28,25 @@ def get_type(arg):
     }
 
     if ieeemac.ismac(arg):
-        return 'mac'
+        potential_types.append("mac")
     #macs can look like hashes, so check them first
     if ishash(arg):
-        return 'hash'
+        potential_types.append("hash")
     ipver = isip(arg)
     if ipver:
-        return ip_types[(ipver, '/' in arg)]
+        potential_types.append(ip_types[(ipver, "/" in arg)])
 
     parts = arg.split(':')
     if len(parts) == 2 and parts[1].isdigit():
-        return 'hostport'
+        potential_types.append("hostport")
 
     if '://' in arg:
-        return 'url'
+        potential_types.append("url")
 
     if '.' in arg:
-        return 'hostname'
+        potential_types.append("hostname")
     
-    return 'username'
+    potential_types.append("username")
```

Comment:
need to return potential_types here

Reply:
I'm already doing so, check out https://github.com/JordanBarnartt/ninfo/blob/6f46517b1e87c826daa9dd26e55fcf6639153c29/ninfo/util.py#L51.

This is just the diff looking weird because the "username" variable is reused.